### ISSUE TITLE
App crash when open Status after it was put on background with Android hardware 'Back' button

### ISSUE THREAD

**Issue by Serhy** [status-im/status-react#8708]
***Type***: Bug

***Summary***: When user taps upon Android hardware 'Back' button being on Chats main view Status is put on the background. However, opening Status again results in an app crash with error `java.lang.illegalArgumentException: No view found for id 0x5ff (unknown) for fragment ScreenFragment{478dd37 #0 id=0x5ff}` 

<img width="352" alt="Screenshot 2019-08-06 at 12 05 50" src="https://user-images.githubusercontent.com/8749671/62527372-aca00a80-b843-11e9-9ed8-b90446eeea95.png">

#### Expected behavior

User end up on Chats view after opening Status if it was put on background with Android hardware 'Back' button before

#### Actual behavior

Status crashes after open if it was put on background with Android hardware 'Back' button before

### Reproduction

- Open Status and create new account
- Staying on Chats main view tap Android hardware `Back` button
- Open Status from the list of running apps (or via Status icon in the list of apps)

### Additional Information

- Status version: Release/Develop 0.14 (
- Operating System: Android

#### Logs

```
08-06 13:23:22.738   354   354 W ResourceType: No package identifier when getting name for resource number 0x00000629
08-06 13:23:22.739   354   354 E FragmentManager: No view found for id 0x629 (unknown) for fragment ScreenFragment{dcd9ea5 #0 id=0x629}
08-06 13:23:22.739   354   354 E FragmentManager: Activity state:
08-06 13:23:22.739   354   354 D FragmentManager:   Local Activity 86d8a26 State:
08-06 13:23:22.740   354   354 D FragmentManager:     mResumed=true mStopped=false mFinished=false
08-06 13:23:22.740   354   354 D FragmentManager:     mChangingConfigurations=false
08-06 13:23:22.740   354   354 D FragmentManager:     mCurrentConfig={0 1.0 themeSeq = 0 showBtnBg = 0 255mcc1mnc [en_US,ru_UA,uk_UA] ldltr sw411dp w411dp h707dp 420dpi nrml long port finger -keyb/v/h -nav/h appBounds=Rect(0, 0 - 1080, 1920) s.10}
08-06 13:23:22.741   354   354 D FragmentManager:     threadConfig={0 1.0 themeSeq = 0 showBtnBg = 0 255mcc1mnc [en_US,ru_UA,uk_UA] ldltr sw411dp w411dp h707dp 420dpi nrml long port finger -keyb/v/h -nav/h appBounds=Rect(0, 0 - 1080, 1920) s.10}
08-06 13:23:22.741   354   354 D FragmentManager:     mLoadersStarted=true
08-06 13:23:22.741   354   354 D FragmentManager:     Active Fragments in f81027a:
08-06 13:23:22.742   354   354 D FragmentManager:       #0: ReportFragment{d0b6a2b #0 android.arch.lifecycle.LifecycleDispatcher.report_fragment_tag}
08-06 13:23:22.742   354   354 D FragmentManager:         mFragmentId=#0 mContainerId=#0 mTag=android.arch.lifecycle.LifecycleDispatcher.report_fragment_tag
08-06 13:23:22.742   354   354 D FragmentManager:         mState=5 mIndex=0 mWho=android:fragment:0 mBackStackNesting=0
08-06 13:23:22.742   354   354 D FragmentManager:         mAdded=true mRemoving=false mFromLayout=false mInLayout=false
08-06 13:23:22.742   354   354 D FragmentManager:         mHidden=false mDetached=false mMenuVisible=true mHasMenu=false
08-06 13:23:22.742   354   354 D FragmentManager:         mRetainInstance=false mRetaining=false mUserVisibleHint=true
08-06 13:23:22.747   354   354 D FragmentManager:         mFragmentManager=FragmentManager{f81027a in HostCallbacks{777e88}}
08-06 13:23:22.747   354   354 D FragmentManager:         mHost=android.app.Activity$HostCallbacks@777e88
08-06 13:23:22.747   354   354 D FragmentManager:         Child FragmentManager{9e8021 in ReportFragment{d0b6a2b}}:
08-06 13:23:22.747   354   354 D FragmentManager:           FragmentManager misc state:
08-06 13:23:22.747   354   354 D FragmentManager:             mHost=android.app.Activity$HostCallbacks@777e88
08-06 13:23:22.747   354   354 D FragmentManager:             mContainer=android.app.Fragment$1@90ccc46
08-06 13:23:22.747   354   354 D FragmentManager:             mParent=ReportFragment{d0b6a2b #0 android.arch.lifecycle.LifecycleDispatcher.report_fragment_tag}
08-06 13:23:22.747   354   354 D FragmentManager:             mCurState=5 mStateSaved=false mDestroyed=false
08-06 13:23:22.747   354   354 D FragmentManager:     Added Fragments:
08-06 13:23:22.747   354   354 D FragmentManager:       #0: ReportFragment{d0b6a2b #0 android.arch.lifecycle.LifecycleDispatcher.report_fragment_tag}
08-06 13:23:22.747   354   354 D FragmentManager:     FragmentManager misc state:
08-06 13:23:22.747   354   354 D FragmentManager:       mHost=android.app.Activity$HostCallbacks@777e88
08-06 13:23:22.747   354   354 D FragmentManager:       mContainer=android.app.Activity$HostCallbacks@777e88
08-06 13:23:22.747   354   354 D FragmentManager:       mCurState=5 mStateSaved=false mDestroyed=false
08-06 13:23:22.747   354   354 D FragmentManager:   ViewRoot:
08-06 13:23:22.747   354   354 D FragmentManager:     mAdded=true mRemoved=false
08-06 13:23:22.747   354   354 D FragmentManager:     mConsumeBatchedInputScheduled=true
08-06 13:23:22.747   354   354 D FragmentManager:     mConsumeBatchedInputImmediatelyScheduled=false
08-06 13:23:22.747   354   354 D FragmentManager:     mPendingInputEventCount=0
08-06 13:23:22.747   354   405 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :signals/signal-received
08-06 13:23:22.747   354   354 D FragmentManager:     mProcessInputEventsScheduled=false
08-06 13:23:22.747   354   354 D FragmentManager:     mTraversalScheduled=false    mIsAmbientMode=false
08-06 13:23:22.747   354   354 D FragmentManager:     android.view.ViewRootImpl$NativePreImeInputStage: mQueueLength=0
08-06 13:23:22.747   354   354 D FragmentManager:     android.view.ViewRootImpl$ImeInputStage: mQueueLength=0
08-06 13:23:22.747   354   354 D FragmentManager:     android.view.ViewRootImpl$NativePostImeInputStage: mQueueLength=0
08-06 13:23:22.747   354   354 D FragmentManager:   Choreographer:
08-06 13:23:22.747   354   354 D FragmentManager:     mFrameScheduled=true
08-06 13:23:22.748   354   354 D FragmentManager:     mLastFrameTime=10160105 (58 ms ago)
08-06 13:23:22.748   354   354 D FragmentManager:   View Hierarchy:
08-06 13:23:22.748   354   354 D FragmentManager:     DecorView@b61b7fe[MainActivity]
08-06 13:23:22.748   354   354 D FragmentManager:       android.widget.LinearLayout{4d47e07 V.E...... ........ 0,0-1080,1920}
08-06 13:23:22.748   354   354 D FragmentManager:         android.view.ViewStub{d49fb34 G.E...... ......I. 0,0-0,0 #10201c4 android:id/action_mode_bar_stub}
08-06 13:23:22.748   354   354 D FragmentManager:         android.widget.FrameLayout{141fd5d V.E...... ........ 0,0-1080,1920}
08-06 13:23:22.748   354   354 D FragmentManager:           android.support.v7.widget.FitWindowsLinearLayout{73966d2 V.E...... ........ 0,0-1080,1920 #7f07000c app:id/action_bar_root}
08-06 13:23:22.748   354   354 D FragmentManager:             android.support.v7.widget.ViewStubCompat{930f3a3 G.E...... ......I. 0,0-0,0 #7f070017 app:id/action_mode_bar_stub}
08-06 13:23:22.748   354   354 D FragmentManager:             android.support.v7.widget.ContentFrameLayout{72596a0 V.E...... ........ 0,0-1080,1920 #1020002 android:id/content}
08-06 13:23:22.748   354   354 D FragmentManager:               com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView{61f9667 V.E...... ........ 0,0-1080,1920 #b}
08-06 13:23:22.748   354   354 D FragmentManager:                 com.facebook.react.views.view.ReactViewGroup{ea61259 V.E...... ........ 0,0-1080,1920 #b15}
08-06 13:23:22.748   354   354 D FragmentManager:                   com.facebook.react.views.view.ReactViewGroup{fae7122 V.E...... ........ 0,0-1080,1920 #b13}
08-06 13:23:22.748   354   354 D FragmentManager:                     com.swmansion.rnscreens.ScreenContainer{bb67eed V.E...... ........ 0,0-1080,1920 #b11}
08-06 13:23:22.748   354   354 D FragmentManager:       android.view.View{231e1e I.ED..... ........ 0,0-1080,63 #102002f android:id/statusBarBackground}
08-06 13:23:22.748   354   354 D FragmentManager:   Looper (main, tid 2) {13c5744}
08-06 13:23:22.748   354   354 D FragmentManager:     Message 0: { when=-41ms callback=android.view.Choreographer$FrameDisplayEventReceiver target=android.view.Choreographer$FrameHandler }
08-06 13:23:22.748   354   354 D FragmentManager:     Message 1: { when=+2s839ms callback=com.google.android.gms.measurement.internal.zzw target=com.google.android.gms.internal.measurement.zzdx }
08-06 13:23:22.749   354   354 D FragmentManager:     Message 2: { when=+58m56s657ms callback=com.google.android.gms.measurement.internal.zzw target=com.google.android.gms.internal.measurement.zzdx }



```

**Comment by annadanchenko** [comment]
@rachelhamlin v1 scope

**Comment by rachelhamlin** [comment]
@siphiuel are you working on this issue?

**Comment by vitvly** [comment]
@Serhy Could not reproduce this on latest develop. Both when opening from task list, and via clicking on the app icon.

**Comment by Serhy** [comment]
@siphiuel that strange. Reproducible for me on real devices with Android 8.1 and Android 9.
Latest develop nightly build: https://status-im.ams3.digitaloceanspaces.com/StatusIm-190829-025900-7c7eb0-nightly.apk

![1231233dded](https://user-images.githubusercontent.com/8749671/63937893-33738c00-ca6c-11e9-80a7-e5a29b84b7ac.gif)

**Comment by vitvly** [comment]
@Serhy Could reproduce it on a real device using the nightly as well! I've tried on emulator before. However, it does not happen to me always - sometimes clicking on an app from task list goes well without errors.

Pushed a tentative "fix" for this in https://github.com/status-im/status-react/pull/8868. Seems that it is caused by the react-native-screens library we use for smoother animations. Will try some alternative approaches.
